[sigmoid] Add canonicalized IR as an option.

zhxchen17 opened the pull request:
Summary: as title, the "canonical" flag is added to sigmoid serializer, so that we can optionally "normalize" the IR to give stable names and orders to IR nodes, which could help with the cases to compare IR definitions.

Test Plan: buck run @//mode/opt //aps_models/ads/config_model_authoring/stability:cli export-generated-module-state-command

Differential Revision: D52431965

avikchaudhuri reviewed (commented):
I'm not sure I follow, where is the `canonical` flag and what does it mean to normalize an IR?

zhxchen17 committed:
[sigmoid] Add canonicalized IR as an option. (#116758)

Summary:

as title, the "canonical" flag is added to sigmoid serializer, so that we can optionally "normalize" the IR to give stable names and orders to IR nodes, which could help with the cases to compare IR definitions.

Test Plan: buck run @//mode/opt //aps_models/ads/config_model_authoring/stability:cli export-generated-module-state-command

Reviewed By: angelayi

Differential Revision: D52431965
